reset_security_server.sh

```diff
@@ -64,12 +65,18 @@ add_public_key() {
     for name in $names
     do
         printf "\nAdding public key to LXD container %s\n" "$name"
-        lxc exec "$name" -- bash -c "mkdir -p $3"
-        lxc exec "$name" -- bash -c "echo \"$(cat "$2")\" > $3/$4"
+        lxc exec "$name" -- bash -c "useradd -c \"$2 user\" -d /home/$2 -s /bin/bash $2"
+        lxc exec "$name" -- bash -c "echo \"$2 ALL=(ALL) NOPASSWD: ALL\" > /etc/sudoers.d/$2"
+        lxc exec "$name" -- bash -c "mkdir -p /home/$2/$4"
+        lxc exec "$name" -- bash -c "echo \"$(cat "$3")\" > /home/$2/$4/$5"
```

Comment:
Replace $N with local variables here and below, ``"echo \"$(cat "$3")\" > /home/$2/$4/$5"``is opaque.